eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -2,16 +2,17 @@ parameters:
   Directory: 'not-specified'
   IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
   
-
 steps:
-  - task: PowerShell@2
-    displayName: Link verification check
-    inputs:
-      pwsh: true
-      workingDirectory: $(Build.SourcesDirectory)/${{ parameters.Directory }}
-      filePath: eng/common/scripts/Verify-Links.ps1 
-      arguments: >
-        -urls $(dir -r -i *.md) 
-        -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" 
-        -recursive: $false 
-        -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
\ No newline at end of file
+- task: PowerShell@2
+  displayName: Link verification check
+  inputs:
+    pwsh: true
+    workingDirectory: $(Build.SourcesDirectory)/${{ parameters.Directory }}
+    filePath: eng/common/scripts/Verify-Links.ps1 
+    arguments: >
+      -urls $(dir -r -i *.md) 
+      -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" 
+      -recursive: $false 
+      -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
+      -branchReplaceRegex "(https://github.com/.*/blob/)master(/.*)"
```

Comment:
We should update this to `($env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI.*/blob/)master(/.*)`